static/randomizer.js

```diff
@@ -0,0 +1,35 @@
+
+(function (){
```

Comment:
Don't thing we should add unuseful names into global scope. I created it to hide `items`. Can be moved into json later

Replaced rundomazer class into object. 
Also we can move it into utils as a function. But need to find place for tips before (json file, translates or settings)